stable/kommander-thanos/values.yaml

```diff
@@ -0,0 +1,53 @@
+# Default values for kommander-thanos.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Internal address for the cluster's Thanos gRPC service.
+# thanosAddress: "HOST:PORT"
+thanosAddress: "prometheus-kubeaddons-prom-prometheus.kubeaddons.svc.cluster.local:10901"
+
+federate:
+  systemNamespace:
+    name: kommander-system
```

Comment:
Is there a way to set this value using the kommander chart's `federate.systemNamespace.name`? I don't like duplicating it here, but I couldn't figure out how else to set the value.

Reply:
This should happen through kubeaddons, somehow... This will be set by kommander chart (assuming this is going to be a dependency of kommander chart?) and so will be passed down from there to this chart when installing kommander chart.

Reply:
yeah, this should be defined by kubeaddons-config properties

Reply:
the `Addon` will take care of the namespace, I believe this field can be removed and the release-namespace can be used instead.

Reply:
> the Addon will take care of the namespace, I believe this field can be removed and the release-namespace can be used instead.

if it's a sub chart, that's only if the `kommander-system` matches the release namespace of the commander addon, which it doesn't:

https://github.com/mesosphere/kubeaddons-configs/blob/master/templates/kommander.yaml#L14

Reply:
okay, I understand the reason. We can keep it as it is because the addon will set a diff namespace value, it isn't very clean but it is valid for this scenario.

Reply:
took a look for an alternative, but: https://github.com/helm/helm/issues/2813